src/router/Multicall.sol

```diff
@@ -0,0 +1,34 @@
+// forked from https://github.com/Uniswap/v3-periphery/blob/main/contracts/base/Multicall.sol
```

Comment:
Is this an exact copy?

Reply:
think so

Reply:
guess i removed the interface andd added the comment to the function